definitions/pipelines/germline_exome.cwl

```diff
@@ -186,7 +186,7 @@ steps:
             contamination_fraction: extract_freemix/freemix_score
             vep_cache_dir: vep_cache_dir
             synonyms_file: synonyms_file
-            annotate_coding_only: annotate_coding_only
+            coding_only: annotate_coding_only
```

Comment:
As of a few versions ago, DV takes `annotate_coding_only` as the param, not just `coding_only`
https://github.com/genome/analysis-workflows/blob/master/definitions/pipelines/detect_variants.cwl#L72

Your above detect_variants.cwl file has it as well - old branch?

Reply:
That input goes to `germline_detect_variants.cwl`:
https://github.com/genome/analysis-workflows/blob/afdce1af762f593ad21bc4036483c792ffbc1559/definitions/subworkflows/germline_detect_variants.cwl#L32-L33

Reply:
ah, my mistake!   Maybe we should harmonize those params?  (I prefer `annotate_coding_only` as being more explicit about what it actually controls)

Reply:
Sounds good to me--changed this input name everywhere (except the VEP tool itself).